lang/po/Firmware_fr.po

```diff
@@ -134,24 +134,24 @@ msgstr "Retour"
 #: ../../Firmware/Marlin_main.cpp:4679 ../../Firmware/messages.cpp:12
 #: ../../Firmware/ultralcd.cpp:1365 ../../Firmware/ultralcd.cpp:4167
 msgid "Bed"
-msgstr "Plateau"
+msgstr "Lit"
```

Comment:
Plateau is more widely used in French than "Lit"

Reply:
I know and we have to stick with `lit` at the moment due to char limitations and French using most space.